spec/tapioca/compilers/symbol_table_compiler_spec.rb

```diff
@@ -2712,6 +2712,8 @@ def c
       RUBY
 
       output = template(<<~RBI)
+        ::Foo::FooAttachedClass = Foo::FooAttachedClass
```

Comment:
Extraneous constant definition

Reply:
Ah, I think I know why this might be. I'll have a look at this tomorrow.